index.html

```diff
@@ -1522,6 +1522,11 @@ <h3 id="navigation-scope-security-considerations">
           <code>background_color</code> member</a> with <var>manifest</var> as
```

Comment:
Guess you could add yourself as editor to this patch? @marcoscaceres ?

Reply:
@kenchris I'd be honored to do so; but do I need any sort of formal approval from the editors and/or WG Chairs?

Reply:
As long as you are approved to actually contribute to the spec, ie WG member etc, I don't see why Marcos and I, couldn't just add/nominate you. Maybe @marcoscaceres would know,

Reply:
@RobDolinMS you don't need any formal approval.

Reply:
@kenchris let's do this separately. @RobDolinMS will need to find his W3C ID, etc.